source/Pacioli/Pacioli.Lib/Models/JournalEntry.cs

```diff
@@ -35,15 +39,37 @@ public record JournalEntry
 
             EnsureDebitsAndCreditsBalance(debits, credits);
 
+            UserId = userId;
             Date = date;
             Debits = new List<JournalEntryDebitLine>(debits);
             Credits = new List<JournalEntryCreditLine>(credits);
         }
 
+        public string UserId { get; }
         public DateTime Date { get; }
         public string Description { get; }
         public List<JournalEntryDebitLine> Debits { get; }
         public List<JournalEntryCreditLine> Credits { get; }
+        public bool Closed { get; private set; }
+
+        public ImmutableList<Review> Reviews { get; private set; }
+
+        public void AddReview(Review review)
+        {
+            if (Closed)
+                throw new InvalidOperationException("Journal is already approved and closed");
+
+            if (string.Compare(UserId, review.Reviewer, true) == 0)
```

Comment:
Strings use value equality semantics, why are we using .Compare instead of `==` or .Equals?